### ISSUE TITLE
Stuck with application menu

### ISSUE THREAD

**Issue by benintech** [G-dH/advanced-alttab-window-switcher#33]
Maybe it's just me missing something here, but :

- Pressing the \<Menu\> key (the key usually on the right of the space bar, somewhere between \<AltGr\> and \<Ctrl\>) displays the application menu of the selected item :
![application-menu-stuck](https://user-images.githubusercontent.com/5140169/183280933-5659f336-9f7f-40e4-adef-2bfc39484597.png)

- But then there's no way of getting rid of that menu, except pressing \<Esc\> which closes AltTab altogether (or using the mouse right-click)

I think pressing the \<Menu\> key again should close it if it's open. The menu key should be a toggling key

**Comment by G-dH** [comment]
In the last official version I've changed Esc key behavior - replaced press signal with the release signal because of full-screen youtube video reacting on the release event of the Ecs key. Almost immediately after I released it on e.g.o I realized that this change negatively affects navigation inside AATWS and I changed it back. Try to install the latest repository version.
When the context menu is open, AATWS doesn't control the key handling, it would need to hack the menu.
